src/inc/common/Mlp_Helpers.php

```diff
@@ -478,8 +478,16 @@ public static function show_linked_elements( $args ) {
 		}
 
 		$output .= '</ul></div>';
-
-		return apply_filters( 'show_linked_elements', $output, $items, $params );
+		
+		/**
+		 * Filter the output of the linked elementes
+		 * 
+		 * @param	string $output the generated HTML
```

Comment:
Redundant spaces after `@param`s.
Descriptions should start with capitals and end with full stops.